docs/devtools-throttling-deprecation.md

```diff
@@ -0,0 +1,9 @@
+# DevTools Throttling Deprecation Notice
+
+Starting with Chrome 80, Lighthouse within Chrome DevTools is deprecating the throttling configuration. To wit:
+
+1. `Simulated Throttling` will become the only option
+1. For awhile, you may also use `Applied Throttling` by unchecking the `Simulated Throttling` checkbox
```

Comment:
fwiw a radio button seems less confusing for this. If "Applied Throttling" is unchecked I would assume it's not throttled, personally